### ISSUE TITLE
Support for Radshid X0 device

### ISSUE THREAD

**Issue by 0x7a657573** [traccar/traccar#5235]
**Is your feature request related to a problem? Please describe.**
unfortunately traccar don't support and device of Radshid co, I Want to add some devices of this company to traccar  

**Describe the solution you'd like**
 I would like traccar to support the [Radshid](https://radshid.com/product-category/gps/) devices.

**Describe alternatives you've considered**
 I did not find any platforms that support the Radshid devices.

how can I add radshid devices to traccar ? 
I want tried to get protocol from company, after that you can added this devices to your platform ?

**Comment by tananaev** [comment]
Do you have the protocol documentation?

**Comment by 0x7a657573** [comment]
Yes, after correspondence with the company, I finally got the X0 protocol 
[X0-Protocol-en.pdf](https://github.com/traccar/traccar/files/13659651/X0-Protocol-en.pdf)

**Comment by tananaev** [comment]
Thanks. We'll keep this ticket open for now to gauge the interest from the community. In case you're interested in sponsoring the work, send us an email to discuss.

**Comment by 0x7a657573** [comment]
Can I add this protocol to traccar and make PR ?
If yes, where should I start?

**Comment by tananaev** [comment]
You can. We have a documentation available on the website.

**Comment by 0x7a657573** [comment]
finally I added radshid x0 protocol to traccar :)
In [this](https://github.com/0x7a657573/traccar) repository, can I merge my repository to traccar ?

**Comment by tananaev** [comment]
Please create a PR and we'll check.

**Comment by 0x7a657573** [comment]
radshid x0 has many event like alarm, SOS, jamming, time over, angle , .....
how can add this event to traccar?

**Comment by MASOODOK** [comment]
> Yes, after correspondence with the company, I finally got the X0 protocol [X0-Protocol-en.pdf](https://github.com/traccar/traccar/files/13659651/X0-Protocol-en.pdf)
good work- Its hard to find any information from Persian company! i tray many time  to find AFRA gps protocol but I did not succeed !
